01-Introduction.Rmd

```diff
@@ -1,8 +1,83 @@
+
+3. [Search existing bug reports](https://bugs.r-project.org/query.cgi)
+
+A [quick search](https://bugs.r-project.org/page.cgi?id=quicksearch.html) bar is available on the home page where you can enter a bug # to search or some search terms.
+
+![Screenshot of the quick search bar on the home page of Bugzilla.](img/quicksearch.png)
+
+There is also a section for Common Queries on the home page which includes links to bugs reported and changed in the last 24 hours and last 7 days.
+
+![Screenshot of the Common Queries section on the home page of Bugzilla.](img/commonquery.png)
+
+## Quick start to the guide
+
+The guide is intended to be a comprehensive resource for contributing to base R. There are a number of chapters and chapter sections devised to help with the process of contributing to base R:
+
+1. The [Getting Started](#GetStart) covers the instructions on how to install R in Windows and the tools required to build R and R packages in Windows. 
+
+<!-- TODO(Saranjeet): Add other operating systems as they get included in the guide-->
+
+2. The [Bug Tracking](#BugTrack) and the [Reviewing Bugs](#ReviewBugs) chapters discuss how to find bugs in R and how to review bug reports that are submitted to Bugzilla.
+
+3. The [Finding the Source](#FindSource) chapter discusses how to have an overview of the R codebase like finding the implementation of a base function written in R and/or the primitive implementation written in C.
+
+4. The [Lifecycle of a Patch](#FixBug) chapter discusses how to create a patch to propose a bug fix.
+
+5. The [Documenting R](#Doc) chapter describes the styleguide for R's documentation, how to document for R, and report and review bugs (and/or suggest corrections for them) in the existing documentation.
+
+6. How to test pre-release versions of R is discussed in the chapter on [Testing Pre-release R Versions](#TestRVer).
+
+7. The [R Core Developers](#CoreDev) chapter contains a list of the former and current members of the R Core team who have write access to the R source.
+
+8. To get more information on how to find assistance to resolve issues faced while working on R, refer to the [Where to Get Help](#WhereToGetHelp) chapter.
+
+9. To keep up with the developments in R refer to some of the resources available in the [News and Announcements](#News)
+
+10. Resources and tools that may be relevant for R developers are available in the [Developer Tools](#DevTools) chapter.
+
+
+## Overview of different ways of contributing to base R Project
+
+One can contribute to the base R project in a number of different ways. Some of them are listed below:
```

Comment:
```suggestion
Contributions to base R are possible in a number of different ways. Some of them are listed below:
```

Reply:
I think this overview should come much earlier, after the introductory paragraph. This is what new contributors need to know right at the start: the main ways that the can contribute to the R project and which chapter to jump to if they want to get started on a particular topic.

Then you could have the more detailed overview of every chapter (for those that want to find out more about the big picture before jumping in).